.github/workflows/mkosi.yml

```diff
@@ -186,21 +187,25 @@ jobs:
 
       - name: Configure meson
         run: |
-          meson setup build \
-              --buildtype=debugoptimized \
-              -Dintegration-tests=true \
-              -Dremote=enabled \
-              -Dopenssl=enabled \
-              -Dblkid=enabled \
-              -Dtpm2=enabled \
-              -Dlibcryptsetup=enabled \
-              -Dlibcurl=enabled \
-              -Drepart=enabled \
-              -Dfirstboot=true \
-              -Dsysusers=true \
-              -Dtmpfiles=true \
-              -Dhwdb=true \
+          OPTIONS=(
+              --buildtype=debugoptimized
+              -Dintegration-tests=true
+              -Dremote=enabled
+              -Dopenssl=enabled
+              -Dblkid=enabled
+              -Dtpm2=enabled
+              -Dlibcryptsetup=enabled
+              -Dlibcurl=enabled
+              -Drepart=enabled
+              -Dfirstboot=true
+              -Dsysusers=true
+              -Dtmpfiles=true
+              -Dhwdb=true
               -Dvmspawn=enabled
+          )
+          [ -n ${{ matrix.sanitizers }} ] && OPTIONS+=( -Dservice-watchdog= )
```

Comment:
Is this for #35335?
I do not think it is killed by the watchdog, but killed by TimeoutStopSec=, which was extended to 2min by b75befc3c98bcfa57ab801a24234ecab1c0b34a9.

Note, fedora has /usr/lib/systemd/system/service.d/10-timeout-abort.conf , so with the stop timeout, SIGABRT is sent to the service.

So, I think it is better to disable the drop-in 10-timeout-abort.conf when running with the sanitizers.

Hopefully, any failures in stopping services will be caught by sanitizers. So, not necessary to dump core.

Also, the drop-in config is Fedora specific, I guess that's why we do not see similar issue on CentOS CI with sanitizers.

Reply:
-> #35460.

Reply:
Yeah, that PR makes more sense.

The good news is that my syntax change is interpreted correctly:
```
   User defined options
    buildtype                                : debugoptimized
    blkid                                    : enabled
    firstboot                                : true
    hwdb                                     : true
    integration-tests                        : true
    libcryptsetup                            : enabled
    libcurl                                  : enabled
    openssl                                  : enabled
    remote                                   : enabled
    repart                                   : enabled
    service-watchdog                         : 
    sysusers                                 : true
    tmpfiles                                 : true
    tpm2                                     : enabled
    vmspawn                                  : enabled
```

I'll update with just the syntax change, to make future changes to options easier.